decidim-core/lib/decidim/query_extensions.rb

```diff
@@ -15,7 +15,7 @@ def self.extend!(type)
           type !types[ProcessType]
           description "Lists all processes."
 
-          resolve ->(_obj, _args, ctx) {
+          resolve lambda { |_obj, _args, ctx|
```

Comment:
Mmm, are these equivalent? I'm surprised this now shows as uncovered in codecov... Perharps I should just revert this one commit?